ntbs-service/DataMigration/NotificationImportService.cs

```diff
@@ -250,5 +255,57 @@ private async Task<List<string>> FilterOutImportedIdsAsync(PerformContext contex
             }
             return legacyIdsToImport;
         }
+        
+        private async Task MigrateOrUpdateCaseManager(Notification notification,
```

Comment:
I think this method has got a bit too long, and would benefit from being broken up a bit.